### PR TITLE
Attempt to fix some cron jobs

### PR DESCRIPTION
**Pull request by Cadair**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -2,6 +2,7 @@ name: $(BuildDefinitionName)_$(Date:yyyyMMdd)$(Rev:.rr)
 variables:
   CIBW_MANYLINUX_X86_64_IMAGE: manylinux2014
   CIBW_MANYLINUX_I686_IMAGE: manylinux2014
+  JSOC_EMAIL: "jsoc@cadair.com"  # For the example gallery on the dev deps build
```
Comment (nabobalis) [inline azure-pipelines.yml:5]: We are dropping this on the metadata PR.

Comment (Cadair) [reply azure-pipelines.yml:5]: Dropping what?

Comment (nabobalis) [reply azure-pipelines.yml:5]: The gallery build.

Comment (Cadair) [reply azure-pipelines.yml:5]: Well we can pull this in that PR if this gets merged first?

File: azure-pipelines.yml [hunk 2]
```diff
@@ -121,9 +122,6 @@ stages:
               - hdf5-devel
 
           envs:
-            - linux32: py38-32bit
```
Comment (Cadair) [inline azure-pipelines.yml:124]: I cracked and killed it.

File: azure-pipelines.yml (hunk 1 above)
Comment (nabobalis) [reply azure-pipelines.yml:5]: Fair.

File: azure-pipelines.yml (hunk 2 above)
Comment (nabobalis) [reply azure-pipelines.yml:124]: I mean it makes my life easier if I kill it on the other repos too.

Comment (Cadair) [reply azure-pipelines.yml:124]: Only one where it's potentially useful is sunkit-image and pyflact as there is a lot of C

Comment (nabobalis) [reply azure-pipelines.yml:124]: I might keep it around there.

**Review by nabobalis — APPROVED** [review]

**Pull request merged**